### ISSUE TITLE
URL.createObjectURL are currently marked unsafe

### ISSUE THREAD

**Issue by bfanger** [driftyco/ionic#3719]
This is [already fixed](https://github.com/angular/angular.js/pull/4623) in regular angular, but ionic  overwrites the [imgSrcSanitizationWhitelist](https://github.com/angular/angular.js/blob/master/src/ng/sanitizeUri.js) value in [js/angular/service/ionicConfig.js](https://github.com/driftyco/ionic/blob/master/js/angular/service/ionicConfig.js#L632).

To fix this add `|blob` to the regex:
```js
$compileProvider.imgSrcSanitizationWhitelist(/^\s*(https?|ftp|file|blob|ms-appx|x-wmapp0):|data:image\//);
```

**Comment by Ionitron** [comment]
Greetings @bfanger!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3719). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3719)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by mhartington** [comment]
Not sure I follow, this is already in 1.0 and release.

https://github.com/driftyco/ionic/blob/master/js/angular/service/ionicConfig.js#L632

**Comment by bfanger** [comment]
Ah, it was indeed [fixed 8 days ago](https://github.com/driftyco/ionic/commit/d684d62a2418a57db9fc9ce44520114243a2daa5) i was using 1.0.0-rc5,

**Comment by bfanger** [comment]
When using the cordova camera plugin, it uses a `content://` scheme. Which will be marked unsafe: `unsafe:content://com.android.providers.media.documents/document/image%3A47`

Adding `content` to the imgSrcSanitizationWhitelist fixes this issue.

```js
$compileProvider.imgSrcSanitizationWhitelist(/^\s*(https?|tel|ftp|file|blob|content|ms-appx|x-wmapp0):|data:image\//); 
```

**Comment by mhartington** [comment]
This should be fixed with 8572f38118127743a6bd7b3ca0ac29555e1a9985
